Update to Predictive Alerts page (#20501)
newrelic/docs-website: src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx

          When the Predictive Alerts public preview period ends at any point for any reason, your alert conditions evaluation window will only evaluate against the static threshold, which may result in a slight delay to detecting an alert violation.
        </Callout>

        

>         New Relic's <DNT>**Predictive Alerts**</DNT> analyze historical data to predict future trends. When the predicted trend breaches static thresholds, you receive an alert notification, giving you the opportunity to act before disruptions occur.

Reviewer: ```suggestion
        New Relic's <DNT>**Predictive Alerts**</DNT> analyze historical data to predict future trends. If the predicted trend shows that static thresholds maybe breached soon, you receive an alert notification, giving you the opportunity to act before disruptions occur.
```

>         For more information, refer to the [Predictive Alerts documentation](/docs/alerts/create-alert/set-thresholds/predictive-alerts/). If you are already acquainted with the <DNT>**Predictive Alerts**</DNT> feature, perform the following steps to enable it for your alert condition:

Reviewer: ```suggestion
        For more information, refer to the [Predictive Alerts documentation](/docs/alerts/create-alert/set-thresholds/predictive-alerts/). 
        
        To enable predictive alerts for your alert condition, do the following steps:
```

        1. In the <DNT>**Set condition thresholds**</DNT> section, select the threshold condition type as <DNT>**Static**</DNT>.
        2. For the predictive alerts, enable the <DNT>**Predict future behavior**</DNT> toggle.

>         3. Set the look-ahead time to predict how far into the future the threshold might be breached. The maximum look ahead time is 360 times the [window duration](/docs/alerts/create-alert/create-alert-condition/alert-conditions/#window-duration).

Reviewer: ```suggestion
        3. Set the look-ahead time. This indicates how far into the future it should look for predicted breaches. The maximum look ahead time is 360 times the [window duration](/docs/alerts/create-alert/create-alert-condition/alert-conditions/#window-duration).
```
